GenerateAspNetCoreClient.Command/ClientModelBuilder.cs

```diff
@@ -277,7 +302,12 @@ private List<Parameter> GetParameters(ApiDescription apiDescription)
                 parameterName = new string(parameterName.Where(c => char.IsLetterOrDigit(c)).ToArray());
 
                 var type = parameterDescription.Source == BindingSource.FormFile
```

Comment:
I think this nested ternary goes beyond readable. Could you please replace it with separate if statements?

(Question regarding type check applies here as well)

Reply:
Agreed. And it should only require checking IEnumerable.